crates/torii/src/cli.rs

```diff
@@ -29,13 +35,79 @@ mod types;
 #[cfg(test)]
 mod tests;
 
+pub(crate) fn dojo_metadata_from_workspace(ws: &Workspace<'_>) -> Option<DojoMetadata> {
+    Some(ws.current_package().ok()?.manifest.metadata.dojo())
+}
+
+#[derive(Default, Deserialize, Debug, Clone)]
+pub(crate) struct DojoMetadata {
+    env: Option<Environment>,
+}
+
+#[derive(Default, Deserialize, Clone, Debug)]
+pub struct Environment {
+    rpc_url: Option<String>,
+    account_address: Option<String>,
+    private_key: Option<String>,
+    keystore_path: Option<String>,
+    keystore_password: Option<String>,
+    world_address: Option<String>,
+}
```

Comment:
instead of copying we can move this from sozo, to a crate which both `sozo` and `katana` uses. from a quick look there is `dojo-world`. 

katana doesn't depend on `dojo-types` but maybe we should move this to there and take its dependency in katana.

Reply:
make sense, I would maybe put Environment / DojoMetatdata at `dojo-types` and make `sozo` and `katana` and `torii` all dependent on that type